packages/vue-flicking/src/Flicking.ts

```diff
@@ -0,0 +1,147 @@
+/**
+ * Copyright (c) 2015 NAVER Corp.
+ * egjs projects are licensed under the MIT license
+ */
+
+import NativeFlicking, { Plugin, FlickingOptions, withFlickingMethods, DEFAULT_OPTIONS } from "@egjs/flicking";
+import ChildrenDiffer from "@egjs/vue-children-differ";
+import ListDiffer, { DiffResult } from "@egjs/list-differ";
+import { Component, Vue, Prop } from "vue-property-decorator";
+import { CreateElement, VNodeData, VNode } from "vue";
+
+@Component({
+  name: "Flicking",
+  directives: {
+    "children-differ": ChildrenDiffer,
+  },
+})
+export default class Flicking extends Vue {
+  // Tag of wrapper element
+  @Prop({ type: String, default: "div", required: false }) tag!: string;
+  @Prop({ type: Object, default: () => ({}), required: false }) options!: Partial<FlickingOptions>;
+  @Prop({ type: Array, default: () => ([]), required: false }) plugins!: Plugin[];
+
+  // Following decorator will inject native Flicking's method into Vue-Flicking
+  @withFlickingMethods
+  private $_nativeFlicking!: NativeFlicking;
+  private $_pluginsDiffer!: ListDiffer<Plugin>;
+  private $_cloneCount!: number;
+
+  public mounted() {
```

Comment:
@WoodNeck Is there `unmouted()` in Vue? It need to destroy flicking after unmounted.

Reply:
I forgot to add one, thanks for reminding.